reports/upload-completed.1.0.0.schema.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/cdcent/data-exchange-messages/reports/base",
+    "title": "Upload Completed Report",
+    "type": "object",
+    "required": [
+        "content_schema_name",
+        "content_schema_version",
+        "status"
+    ],
+    "properties": {
+        "content_schema_name": {
+            "type": "string"
+        },
+        "content_schema_version": {
+            "type": "string"
+        },
+        "status": {
```

Comment:
Can the upload started and completed status be made an enum?  ex:
"aggregation": {
                    "type": "string",
                    "enum": ["SINGLE", "BATCH"],
                    "description": "Enumeration: [SINGLE, BATCH]."
                },